sdk/python/foundation-models/system/finetune/chat-completion/chat-completion.ipynb

```diff
@@ -0,0 +1,731 @@
+    "\n",
+    "The original datasets consists of 1.4M dialogues generated by ChatGPT and spanning a wide range of topics. To create UltraChat 200k, the following logic is applied:\n",
+    "\n",
+    "* Selection of a subset of data for faster supervised fine tuning.\n",
+    "* Truecasing of the dataset, as we observed around 5% of the data contained grammatical errors like \"Hello. how are you?\" instead of \"Hello. How are you?\"\n",
+    "* Removal of dialogues where the assistant replies with phrases like \"I do not have emotions\" or \"I don't have opinions\", even for fact-based prompts that don't involve either.\n",
+    "\n",
+    "**Dataset Structure**\n",
+    "\n",
+    "The dataset has four splits, suitable for:\n",
+    "* Supervised fine-tuning (sft).\n",
+    "* Generation ranking (gen) via techniques like rejection sampling or PPO.\n",
+    "The number of examples per split is shown as follows:\n",
+    "\n",
+    "| train_sft | test_sft | train_gen | test_gen |\n",
+    "| :- | :- | :- | :- |\n",
+    "| 207865 | 23110 | 256032 | 28304 |\n",
+    "\n",
+    "\n",
+    "The dataset is stored in parquet format with each entry using the following schema:\n",
+    "``` json\n",
+    "{\n",
+    "    \"prompt\": \"Create a fully-developed protagonist who is challenged to survive within a dystopian society under the rule of a tyrant. ...\",\n",
+    "    \"messages\":[\n",
+    "        {\n",
+    "            \"content\": \"Create a fully-developed protagonist who is challenged to survive within a dystopian society under the rule of a tyrant. ...\",\n",
+    "            \"role\": \"user\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"content\": \"Name: Ava\\n\\n Ava was just 16 years old when the world as she knew it came crashing down. The government had collapsed, leaving behind a chaotic and lawless society. ...\",\n",
+    "            \"role\": \"assistant\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"content\": \"Wow, Ava's story is so intense and inspiring! Can you provide me with more details.  ...\",\n",
+    "            \"role\": \"user\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"content\": \"Certainly! ....\",\n",
+    "            \"role\": \"assistant\"\n",
+    "        }\n",
+    "    ],\n",
+    "    \"prompt_id\": \"d938b65dfe31f05f80eb8572964c6673eddbd68eff3db6bd234d7f1e3b86c2af\"\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "### Model\n",
+    "We will use the [Phi-3-mini-4k-instruct](https://ml.azure.com/registries/models-staging/models/Phi-3-mini-128k-instruct/version/15?flight=itpmerge&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview) model to show how user can finetune a model for chat-completion task. If you opened this notebook from a specific model card, remember to replace the specific model name.\n",
```

Comment:
Is the flight needed? _itpmerge_?

Reply:
discussed with @amangupta26 we will be merging the notebook now itself, we can get rid of model card hyperlink for now and have it later once model get published if needed